### DISCUSSION TITLE
How to configure super-linter in my pipeline to ignore only certain terrascan rules?

### DISCUSSION THREAD

**Discussion by atorcode** [super-linter/super-linter#5786] [Q&A]
I followed the [terrascan documentation](https://github.com/tenable/terrascan?tab=readme-ov-file) for how to disable a specific rule, and ran my pipeline again, but my ignore comment isn't being honored. Firstly, am I understanding the method to ignore rules correctly? Here's an example:

```
#"ts:skip=AWS.DynamoDb.Logging.Medium.007 PITR not needed for state lock table
resource "aws_dynamodb_table" "terraform_state_lock" {
  name         = "terraform-state-lock"
  billing_mode = "PAY_PER_REQUEST"
  hash_key     = "LockID"
  attribute {
    name = "LockID"
    type = "S"
  }
  server_side_encryption {
    enabled = true
  }
}
```
Is there something else that needs to be done in addition to this? Thanks!

**Comment by ferrarimarco** [comment]
Hi! I think you have a typo in your comment.

We don't change anything in this regard, so I would refer to the terrascan docs: https://runterrascan.io/docs/usage/in-file_instrumentation/#rule-skipping
